File: app/src/main/java/com/example/firstproject/MainActivity.java
package com.example.firstproject;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    private Button btn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main); // Pastikan layout ini ada

        btn = findViewById(R.id.button); // ← titik koma diperbaiki

        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, loginactivity.class);
                startActivity(intent); // ← tambahkan ini
            }
        });
    }
}

File: app/src/main/java/com/example/pemrograman4-UMB/MainActivity.kt
package com.example.pemrograman4umb

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    private lateinit var tvWelcome: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        tvWelcome = findViewById(R.id.tvWelcome)
        tvWelcome.text = "Selamat datang di MainActivity!"
    }
}
